examples/atmajs/package.json

```diff
@@ -1,12 +1,12 @@
 {
   "private": true,
   "dependencies": {
-    "atma-class": "^1.0.68",
-    "includejs": "^0.9.10",
+    "atma-class": "^1.1.69",
+    "includejs": "^0.9.14",
     "jquery": "^2.1.3",
-    "maskjs": "^0.10.1",
-    "ruta": "^0.1.12",
-    "todomvc-common": "^1.0.1",
-    "todomvc-app-css": "^1.0.0"
+    "maskjs": "^0.51.26",
+    "ruta": "^0.1.16",
+    "todomvc-app-css": "^1.0.0",
```

Comment:
Now that we're rewriting anyways, mind switching to 2.0? You'll need to update from [ids to classes](https://github.com/tastejs/todomvc-app-css/commit/a722ad8119d559228cba1e4895dabacffffcdb2b). It will make all tests fail but don't worry, we're on it. Thanks!